website/source/docs/secrets/mssql/index.html.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: "docs"
+page_title: "Secret Backend: mssql"
+sidebar_current: "docs-secrets-mssql"
```

Comment:
In fact you can try running the local website server and actually test this to avoid surprises.

Have `Ruby 2.x`, and `bundler` installed.
Then run the following:
cd website
`bundle`
`bundle exec middleman server`